exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -120,7 +120,7 @@ drill.exec: {
     affinity.factor: 1.2
   },
   sys.store.provider: {
-    class: "org.apache.drill.exec.store.sys.zk.ZkPStoreProvider",
+    class: "org.apache.drill.exec.store.sys.store.provider.ZookeeperPersistentStoreProvider",
```

Comment:
The new patch includes a deprecated ZkPStoreProvider that is to be removed in 1.7. Now old configuration should work as is. We should also state this out in 1.6 release notes. Let me know if you guys have any other concerns as for compatibility.